core/includes/install.inc

```diff
@@ -992,18 +992,24 @@ function backdrop_install_system() {
  *   all modules in this list have already been disabled before this function
  *   is called.
  * @param bool $uninstall_dependents
- *   (optional) If TRUE, the function will check that all modules which depend
- *   on the passed-in module list either are already uninstalled or contained in
- *   the list, and it will ensure that the modules are uninstalled in the
- *   correct order. This incurs a significant performance cost, so use FALSE if
- *   you know $module_list is already complete and in the correct order.
- *   Defaults to TRUE.
+ *   (optional)
+ *   - If TRUE, dependency-checking is enabled. This means that any modules
+ *     dependent on the passed-in module(s) are checked to see if they're
+ *     already uninstalled, or if they're included in $module_list. If the
+ *     latter, they will all be uninstalled in the correct order. If, however,
```

Comment:
I know people that have trouble with the phrase "if the latter". And I have trouble remembering what the "latter" is sometimes when it's highly technical, so could we rephrase this from:

> If the latter, they will all be uninstalled in the correct order

Into:

> If all dependents are in the $module_list, they will all be uninstalled in the correct order